packages/plugin-react/src/react.ts

```diff
@@ -77,6 +40,8 @@ export const applyBasicReactSupport = (api: RsbuildPluginAPI) => {
       return;
     }
 
+    chain.resolve.alias.set('react-refresh', REACT_REFRESH_DIR_PATH);
```

Comment:
before this alias will be added by rspack, in 0.5 react-refresh will be a peerDependencies of @rspack/core, i'm not sure whether user need to install react-refresh by themselves when using rsbuild, but I saw react-refresh is **dependencies** of the plugin-react, so I add this alias at here

it's ok to remove this alias if the original intention is let user install react-refresh by themselves

Reply:
`@rsbuild/plugin-react` already provides `react-refresh`, usually users do not need to install it 😄